src/Tribe/Context/locations.php

```diff
@@ -76,6 +76,25 @@
 	],
 	'post_type' => [
 		'read' => [
+			Tribe__Context::FUNC        => static function() {
+				$post_type_objs = get_post_types( [
+					'public' => true,
+					'_builtin' => false,
+				], 'objects' );
+
+				foreach( $post_type_objs as $post_type ) {
+					if ( empty( $post_type->query_var ) ) {
+						continue;
+					}
+
+					$url_value = tribe_get_request_var( $post_type->query_var, false );
+					if ( empty( $url_value ) ) {
+						continue;
+					}
+
+					return $post_type->name;
+				}
```

Comment:
For the context to work correctly, here the code should return `Tribe__Context::NOT_FOUND` when the value we're trying to fetch is not found.